### DISCUSSION TITLE
New Yuuki-Asuna x265 build

### DISCUSSION THREAD

**Discussion by stax76** [staxrip/staxrip#406] [General]
There is a new Yuuki-Asuna x265 build:

x265-Asuna-3.3+10-gf7ef79cce+34.7z

Link to the cloud drive is in the wiki:

https://github.com/staxrip/staxrip/wiki/x265#yuuki-asuna

Here is the announcement:

https://github.com/msg7086/x265-Yuuki-Asuna/issues/11#issuecomment-751247349

**Comment by fmohican** [comment]
Hohoho Asuna Mod, nice!

Thank for providing one of the best encoding tool.

**Reply by stax76** [reply]
I hope it gets soon updated to version 3.4.

**Reply by stax76** [reply]
I've asked why they are not updating to 3.4, unfortunately there is a bug that prevents it.

**Reply by fmohican** [reply]
No problem, no bug(s) is eternal ^^ It will patch

**Reply by ghost** [reply]
The latest build including Patman86's efforts is a great idea!
